tools/scripts/build/build-mobile-curriculum.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs');
+const path = require('path');
+
+exports.buildMobileCurriculum = function buildMobileCurriculum(json) {
+  const mobileStaticPath = path.resolve(__dirname, '../../../client/static');
+
+  if (!fs.existsSync(`${mobileStaticPath}/mobile`)) {
+    fs.mkdir(`${mobileStaticPath}/mobile`, err => {
+      return err;
+    });
+    writeJson(json);
+  } else {
+    writeJson(json);
+  }
+
+  function writeJson(curriculum) {
+    curriculum = JSON.parse(curriculum);
+
+    const superBlocks = Object.keys(curriculum);
+
+    const superBlockObj = { superblocks: superBlocks.slice(1) };
```

Comment:
```suggestion
    const superBlocksAndCertifications = Object.keys(curriculum);

    const superBlockObj = { superblocks: superBlocksAndCertifications.slice(1) };
```
I think this is enough to explain why the slice is necessary.